R/clean.R

```diff
@@ -95,10 +95,13 @@ find_meta_stopwords <- function(metadata,
 #' @param add_stopwords A vector of words to be added to the urbanism stopwords
 #' @return A vector of urbanism-related stopwords
 #' @export
-urbanism_stopwords <- function(add_stopwords = NULL) {
-  stop_words <-
-    c(add_stopwords, "city", "urban", "urbanism", "hab", "km")
+subject_stopwords <- function(add_stopwords = NULL, convert_to_regex = TRUE) {
```

Comment:
I gave this function a more generic name to reflect what in literature is called subject-specific stopwords. Now, with this change, I am wondering if we should write a wrapper function urbanism_stopwords or if we should pass the vector of stopwords as an argument in the paper when calling `subject_stopwords()`. What do you think?